src/lib/fastDownward/distanceMap.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { LearningUnit, Skill } from "../types";
 import { DistanceMap } from "./distanceMap";
+import { getTeachingGoals, getRequiredSkills, getSuggestedSkills} from "../compositeLearningUnit"
```

Comment:
feeld weird that you have to import functions from another domain "object" (compositeLearningUnit) in a test, for a different domain object (distanceMap). 
This will also resolve the duplicate function problem with "newLearningUnit"